tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -139,6 +139,7 @@ bgp/test_bgp_allow_list.py:
     conditions:
       - "'t1' not in topo_type"
       - "platform in ['x86_64-8111_32eh_o-r0', 'x86_64-8122_64eh_o-r0', 'x86_64-8122_64ehf_o-r0']"
+      - "release in ['202412']"
```

Comment:
both failures are not seen in current runs. are you still seeing this failure? it doesn't look like a good idea to skip BGP tests.